addons/website/static/src/snippets/s_map/000.js

```diff
@@ -0,0 +1,41 @@
+/** @odoo-module **/
+
+import publicWidget from 'web.public.widget';
+
+publicWidget.registry.Map = publicWidget.Widget.extend({
+    selector: '.s_map',
+
+    /**
+     * @override
+     */
+    start() {
+        if (!this.el.querySelector('.s_map_embedded')) {
+            // The iframe is not found inside the snippet. This is probably due
+            // the sanitization of a field during the save, like in a product
+            // description field.
+            // In such cases, reconstruct the iframe.
+            const dataset = this.$target[0].dataset;
```

Comment:
`this.$target` is a compatibility stuff in public widgets, you can use `this.$el`, or in this case `this.el` directly.

EDIT: you actually use `this.el` already everywhere else :smile: